src/components/pages/contact/contact-form.tsx

```diff
@@ -56,101 +73,62 @@ export const ContactForm: React.FC = () => {
-              required: t<string>('contact.error.email'),
-              pattern: {
-                value: SIMPLE_EMAIL_PATTERN,
-                message: t<string>('contact.error.email-unknown'),
-              },
-            })}
-            error={errors.email}
-            name="email"
-            label={t<string>('contact.email')}
-          />
-        </GridItem>
-        <GridItem>
-          <InputField
-            required={true}
-            inputRef={register('message', {
-              required: t<string>('contact.error.message'),
-            })}
-            error={errors.message}
-            name="message"
-            label={t('contact.message')}
-            type={'text-area'}
-          />
-        </GridItem>
-        <GridItem>
-          <StyledCaptionText>
-            <Sup>*</Sup> {t('contact.mandatory-field')}
-          </StyledCaptionText>
-        </GridItem>
-        <GridItem>
-          {requestStatus === 'pending' && (
-            <SendButton type="submit">
-              <ButtonText>{t('contact.action.send')}</ButtonText>{' '}
-              <RightArrowIcon />
-            </SendButton>
-          )}
-          {(errors.name || errors.email || errors.message) && (
-            <ErrorMessageSend>{t('contact.action.missing')}</ErrorMessageSend>
-          )}
-          {requestStatus === 'success' && (
-            <SentButton type="button">
-              <ButtonText>{t('contact.action.sent')}</ButtonText>{' '}
-              <CheckmarkIcon />
-            </SentButton>
-          )}
-          {requestStatus === 'error' && (
-            <>
-              <Trans i18nKey="contact.action.again-text">
-                Leider gab es einen Fehler. Bitte versuche es noch einmal.
-                Klappt das nicht, schicke deine Nachricht bitte direkt an
-                <Link to="mailto:beep@satellytes.com">beep@satellytes.com</Link>
-              </Trans>
-              <SendButton type="submit">
-                <ButtonText>{t('contact.action.again')}</ButtonText>{' '}
-                <RightArrowIcon />
-              </SendButton>
-            </>
-          )}
-        </GridItem>
-      </Grid>
-    </form>
+    <>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <StyledGrid nested>
+          <GridItem xs={12} md={6}>
+            <TextInput
+              name={'name'}
+              label={t('contact.name')}
+              control={control}
+              rules={{ required: t<string>('contact.error.name') }}
+            />
+          </GridItem>
+          <GridItem xs={12} md={6}>
+            <TextInput
+              name={'email'}
+              label={t('contact.email')}
+              control={control}
+              rules={{
+                required: t<string>('contact.error.email'),
+                pattern: {
+                  value: SIMPLE_EMAIL_PATTERN,
+                  message: t<string>('contact.error.email-undefined'),
+                },
+              }}
+            />
```

Comment:
I think you could create local components at the top of the file like `Email`, `FirstName` and later the form would look super tidy and easy to navigate:

````
<form>
  <Row>
     <FirstName control={control}/>
     <LastName control={control}/>
   </Row>
  <Row>
     <Email control={control}/>
     ...
   </Row>
</form>
```

Reply:
Good suggestion, thank you 🙏